### PR TITLE
Add supported k8s version in the README

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR adds the description of the supported Kubernetes version.
It is decided according to the test result in the following PR.
https://github.com/scalar-labs/helm-charts/pull/155

Please take a look!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,16 @@ This directory contains the following helm charts.
 
 ## Prerequisites
 
-* Kubernetes 1.19+
 * Helm 3.5+
 
+## Supported Kubernetes version
```
Comment (feeblefakie) [inline README.md:18]:
```suggestion
## Supported Kubernetes versions
```

File: README.md [hunk 2]
```diff
@@ -13,9 +13,16 @@ This directory contains the following helm charts.
...
+
+* Kubernetes 1.25
```
Comment (feeblefakie) [inline README.md:20]: Better to be simplified like this?
```
* 1.[21-25].x
```

**Review by feeblefakie — COMMENTED** [review]
Thank you! Left one minor suggestion.

File: README.md (hunk 2 above)
Comment (kota2and3kan) [reply README.md:20]: Thank you for your suggestion!

It is simple and we can maintain it easily. So, I think it is better from perspective of developer/maintainer (i.e., us).
However, I think there is a possibility that the regular expressions is a bit confusing.

The `list` is easy to understand with a quick glance.
So, from the perspective of users, I think it is better to describe supported version as a list.

What do you think?

**Comment by kota2and3kan** [comment]
I updated the description based on the result of the discussion.
Please take a look again!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**